scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/manager/ManagerApplication.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.scheduling.manager;
+
+import com.amazonaws.blox.lambda.LambdaFunction;
+import com.amazonaws.blox.lambda.AwsSdkV2LambdaFunction;
+import com.amazonaws.blox.scheduling.SchedulingApplication;
+import com.amazonaws.blox.scheduling.scheduler.SchedulerInput;
+import com.amazonaws.blox.scheduling.scheduler.SchedulerOutput;
+import com.amazonaws.blox.scheduling.state.ECSStateClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan("com.amazonaws.blox.scheduling.manager")
+public class ManagerApplication extends SchedulingApplication {
```

Comment:
you don't need to extend another configuration class, you can just import it: https://docs.spring.io/spring-javaconfig/docs/1.0.0.M4/reference/html/ch04s03.html

Reply:
Ah. How do you reference beans from the base config in that case? E.g. on line 47 the `scheduler()` bean references `lambdaClient()` from the base class.

Reply:
Looks like you can autowire the instance; but then it's not clear how I'd override the beans from the base class in tests...

Reply:
why do you need to override anything? unit tests shouldn't rely on config and integ tests should use the exact same config so we're not testing something different that's going to go into prod

Reply:
Again, these are in that fuzzy area between unit/integration tests, in that they don't need access to production resources, but they do check that most of our spring config is sane.

Reply:
As mentioned, these are taking the place of the nonexistent integ tests right now, they will fold into those once we have them.